src/main/java/org/sonarsource/scanner/maven/bootstrap/MavenProjectConverter.java

```diff
@@ -385,10 +383,10 @@ private void synchronizeFileSystemAndOtherProps(MavenProject pom, Map<String, St
     MavenUtils.putAll(userProperties, props);
 
     List<File> mainDirs = mainSources(pom);
-    props.put(ScanProperties.PROJECT_SOURCE_DIRS, StringUtils.join(toPaths(mainDirs), SEPARATOR));
+    props.put(ScanProperties.PROJECT_SOURCE_DIRS, MavenUtils.joinAsCsv(toPaths(mainDirs)));
     List<File> testDirs = testSources(pom);
     if (!testDirs.isEmpty()) {
-      props.put(ScanProperties.PROJECT_TEST_DIRS, StringUtils.join(toPaths(testDirs), SEPARATOR));
+      props.put(ScanProperties.PROJECT_TEST_DIRS,  MavenUtils.joinAsCsv(toPaths(testDirs)));
```

Comment:
Let's remove unnecessary spaces
```suggestion
      props.put(ScanProperties.PROJECT_TEST_DIRS, MavenUtils.joinAsCsv(toPaths(testDirs)));
```